src/Controllers/Common/AuthenticationController.cs

```diff
@@ -181,6 +181,9 @@ public class AuthenticationController : Controller {
             return Unauthorized();
         }
 
+        if (viking.GameVersion is null) viking.GameVersion = ClientVersion.GetVersion(apiKey).ToString();
+        if (viking.GameVersion != ClientVersion.GetVersion(apiKey).ToString()) return Unauthorized(); // do not let players log into users from other games
```

Comment:
this should be value (or bitmask) comparison ... simple compare for equality breaks WoJS lands (they use their own `apiKey` and their own `gameVersion` ... this is used when storing some information - like `productData`) and single account for SoD